packages/autorest.python/autorest/codegen/__init__.py

```diff
@@ -96,7 +98,7 @@ def models_mode(self) -> Union[str, bool]:
     def tracing(self) -> bool:
         return self.options.get(
             "tracing",
-            self.show_operations and not self.unbranded,
+            self.show_operations and self.flavor == "azure",
```

Comment:
Given we will do this test a lot, we should have a static function like:
```python
def is_azure_flavor():
   return self.flavor == "azure"
```
This would make the code more readable, and avoid risk of typo like `self.flavor == "aure"` for future code.

Reply:
Make sense.
Updated: In the future, if we want to update the logic, just need to search `is_azure_flavor`